packages/components/src/navigation/NavTab.tsx

```diff
@@ -67,6 +67,12 @@ const NavTab = memo(
               data-testid={`btn-nav-tab-${title}`}
               role="tab"
               tabIndex={0}
+              onAuxClick={event => {
```

Comment:
I don't think onAuxClick works in safari.

Reply:
Will take a look into this

Reply:
In that case you probably will need `onMouseUp` or `onMouseDown` (check behavior of the golden-layout tabs) and check `event.buttons===4` I think

Reply:
It's 1 not 4, and you should be able to get it from just the click event I would think.

Reply:
`Tab` component seems to use:

```
this.element.on('click', this._onTabClick);
this.element.on('auxclick', this._onTabClick);
```

@dsmmcken any idea if jQuery is making `auxclick` work in Safari, or is there something else happening here?

Reply:
`button === 1` only checks the last pressed button. `buttons` (plural) the value is 4. The plural is all buttons currently being pressed. So you can verify it's only a middle click instead of multiple buttons being held.

The plural uses a bitmask approach so every combination of buttons has a unique number. Left mouse = 1, right mouse = 2, middle = 4. So then left + right = 3, left + middle = 5

Reply:
Actually if `mouseUp` then `button === 1 && buttons === 0` would indicate the middle mouse button being released without any other buttons currently held. `buttons === 4` would be used on mouse down

Reply:
@mattrunyon  This seems overly complicated. What differentiates this from the `Tab` component implementation?

Reply:
Can you check middle click to close actually works on Safari with golden layout? If so, then jQuery is polyfilling the functionality. If not, then we just never tested it works in Safari

You would want to check both button and buttons on mouseUp because you could start dragging, then middle click on accident and possibly get into a very weird state. You just need to check 2 values, so I'm not sure how it's that complicated

Reply:
Looking into it currently, will update

Reply:
Middle click does work with Safari golden layout.

Reply:
I'm following now. Seems like `mouseUp` event with a `button === 1 && buttons === 0` check is the simplest approximation. Just means that if you click down elsewhere and drag onto the tab and then release it would close the tab, but that doesn't really seem like a concern.

Reply:
Spoke to Matt, pushing changes shortly.